### ISSUE TITLE
Launching services

### ISSUE THREAD

**Issue by CMCDragonkai** [svanderburg/disnix#2]
I've read through the documentation, but there's a part I don't understand.

When using NixOS by itself, in order to have services run and be enabled, I usually have to specify it in `configuration.nix` which deterministically sets up the service file for systemd and compiles/installs the package.

When using disnix, and specifying for a service to run, is it modifying the `configuration.nix` and then doing a `nixos-rebuild`, or is it bypassing the `configuration.nix`. If it is bypassing it, how does it setup new services without making NixOS do a rebuild (especially if all the config directories like `/etc` are read-only). If it isn't bypassing it, why bother with disnix, we could just use DisnixOS which would do the same thing (changing the configuration.nix and running system specific services).

Basically why have dysnomia or nixos when you have DisnixOS? NixOS service configuration already have service files just by specifying them inside the `configuration.nix`. Then one only needs to configure it from DisnixOS, and you'll have services running and activated in the NixOS instances!
